src/jit/inlinepolicy.cpp

```diff
@@ -26,7 +26,19 @@
 
 InlinePolicy* InlinePolicy::GetPolicy(Compiler* compiler, bool isPrejitRoot)
 {
-    // For now, always create a Legacy policy.
+
+#ifdef DEBUG
+
+    // If JitInlineRandomSeed is set, use the RandomPolicy.
+    unsigned seed = JitConfig.JitInlineRandomSeed();
+    if (seed != 0)
+    {
+        return new (compiler, CMK_Inlining) RandomPolicy(compiler, isPrejitRoot, seed);
+    }
```

Comment:
Either way the remainder of the stress modes would still be active, right? 

What I was hoping to do with the RandomPolicy was to assess how robust the rest of the jit is to changes in the inliner (since I plan to make changes). Ideally this means just inlining differently but "plausibly" (at least as far as not creating crazily big methods like STRESS_INLINE does). I guess I can get the more or less the same information even if enabling the RandomPolicy also enables a bunch of non-inline stress but this is not what I'd envisioned.

It does not seem easy right now to just enable one stress mode.